MMM-WeatherChart.js

```diff
@@ -697,6 +751,32 @@ Module.register("MMM-WeatherChart", {
             data: maxTemps,
             yAxisID: "y1",
         });
+        if (this.config.showPressure) {
+        
+		 datasets.push({
+		    label: "Pressure",
+		    borderColor: this.config.colorPressure,
+		    pointBackgroundColor: this.config.colorPressure,
+		    datalabels: {
+		        color: this.config.colorPressure,
+		        align: "top",
+		        offset: this.config.datalabelsOffset,
+		        font: {
+		            weight: this.config.fontWeight,
+		        },
+		        display: this.config.datalabelsDisplay,
+		        formatter: function (value) {
+		            let place = 10 ** self.config.datalabelsRoundDecimalPlace;
+		            let label = Math.round(value * place) / place;
+		            return label;
+		        },
+		    },
+		    data: pressures,
+		    yAxisID: "y3",
```

Comment:
Hide the y3 axis at the far left side.

y1, y2 axis is hide by the below code.
https://github.com/mtatsuma/MMM-WeatherChart/blob/master/MMM-WeatherChart.js#L880-L884

Reply:
Updated in commit: 90dff89e4954dfea69af4b40754f8bb7c519a05a